coalib/collecting/Collectors.py

```diff
@@ -126,9 +126,12 @@ def icollect_bears(bear_dir_glob, bear_globs, kinds, log_printer):
         # pass this later to iglob, we need to escape this.
         bear_dir = glob_escape(bear_dir)
         for bear_glob in bear_globs:
-            for matching_file in iglob(
-                    os.path.join(bear_dir, bear_glob + '.py')):
+            matching_files = iglob(
+                    os.path.join(bear_dir, bear_glob + '.py'))
 
+            matching_files = sorted(matching_files)
```

Comment:
No change in the number of lines modified (2), as you still need to add the trailing `)`.
My version allows for more complicated sorting to be added later if needed.